flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBNativeMetricOptions.java

```diff
@@ -239,10 +249,13 @@ public static RocksDBNativeMetricOptions fromConfig(Configuration config) {
 			options.enableActualDelayedWriteRate();
 		}
 
+		options.setColumnFamilyAsVariable(config.getBoolean(COLUMN_FAMILY_AS_VARIABLE));
+
 		return options;
 	}
 
 	private Set<String> properties;
+	private boolean columnFamilyAsVariable;
```

Comment:
Let's set this to `COLUMN_FAMILY_AS_VARIABLE.defaultValue()` to safeguard against potential future changes.